docker-for-ibm-cloud/administering-swarms.md

```diff
@@ -0,0 +1,266 @@
+* Retrieve your IBM Cloud infrastructure [API username and key](https://knowledgelayer.softlayer.com/procedure/retrieve-your-api-key).
+* Create an SSH key, add the public SSH key to IBM Cloud infrastructure and note its label, and locate the file path of the private SSH key on your machine.
+* Retrieve the Docker EE installation URL that you received in your beta welcome email.
+
+To create a Docker EE for IBM Cloud cluster from the CLI:
+1. Log in to the IBM Cloud CLI. If you have a federated ID, use the `--sso` option.
+   ```bash
+   $ bx login [--sso]
+   ```
+2. Target the IBM Cloud org and space.
+   ```bash
+   $ bx target --cf
+   ```
+3. Review the required variables. Flag variables that are marked `Required` must be provided during the create command. Optional flags are set to the default.
+
+   | Flag variable | Description | Default Value | Required? |
+   | ---- | ----------- | ------------- | --- |
+   | `--sl-user`, `-u` | [Log in to IBM Cloud infrastructure](https://control.softlayer.com/), select your profile, and locate your **API Username** under the API Access Information section. | | Required |
+   | `--sl-api-key`, `-k` | [Log in to IBM Cloud infrastructure](https://control.softlayer.com/), select your profile, and locate your **Authentication Key** under the API Access Information section. | | Required |
+   | `--ssh-label`, `--label` | Your IBM Cloud infrastructure SSH key label for the manager node. To create a key, [log in to IBM Cloud infrastructure](https://control.softlayer.com/) and select **Devices > Manage > SSH Keys > Add**. Copy the key label and insert it here. | | Required |
+   | `--ssh-key` | The path to the SSH key on your local client that matches the SSH key label in your IBM Cloud infrastructure account. | | Required |
+   | `--swarm-name`, `--name` | The name for your swarm and prefix for the names of each node. | d4ic | Required |
+   | `--docker-ee-url` | The Docker EE installation URL associated with your subscription. For beta, you received this in your welcome email. | | Required |
+   | `--manager` | Deploy 1, 3, or 5 manager nodes. | 3 | Optional |
+   | `--workers`, `-w` | Deploy a minimum of 1 and maximum of 10 worker nodes. | 3 | Optional |
+   | `--datacenter`, `-d` | The location (data center) that you deploy the cluster to. Availabe locations are dal12, dal13, fra02, hkg02, lon04, par01, syd01, syd04, tor01, wdc06, wdc07. | wdc07 | Optional |
+   | `--verbose`, `-v` | Enable verbose mode | | Optional |
+   | `--hardware` | If "dedicated" then the nodes are created on hosts with compute instances in the same account. | Shared | Optional |
+   | `--manager-machine-type` | The machine type of the manager nodes: u1c.1x2, u1c.2x4, b1c.4x16, b1c.16x64, b1c.32x128, or b1c.56x242. Higher machine types cost more, but deliver better performance: for example, u1c.2x4 is 2 cores and 4 GB memory, and b1c.56x242 is 56 cores and 242 GB memory. | u1c.2x4 | Optional |
```

Comment:
"Higher" doesn't seem like the right word here. Maybe "More powerful"?

Reply:
Change the colon after "performance" to a period (or at least a semicolon, but I prefer shorter sentences)